rules/prefer-https.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+const getDocumentationUrl = require('./utils/get-documentation-url');
+
+const MESSAGE_ID = 'prefer-https';
+const messages = {
+	[MESSAGE_ID]: 'Prefer `{{replacement}}` over `{{value}}`.'
+};
+
+//	Helper to determine if a string is an unsafe URL.
+const containsHttp = nodeValue => /http:\/\/(www\.)?[\w#%+.:=@~-]{2,256}\.[a-z]{2,4}\b([\w#%&+./:=?@~-]*)/.test(nodeValue);
+
+const create = context => {
+	const sourceCode = context.getSourceCode();
+	const reportPreferHttp = node => {
+		const fixed = node.value.replace('http', 'https');
```

Comment:
What if the comment is `// Foo bar http is good: https://asdasd.com`?

I think just replacing `http` substring is too fragile.